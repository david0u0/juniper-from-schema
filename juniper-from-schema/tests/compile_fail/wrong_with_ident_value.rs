#![allow(dead_code, unused_variables, unused_must_use, unused_imports)]
include!("../setup.rs");

use juniper::ID;

graphql_schema_from_file!(
    "../../../juniper-from-schema/tests/schemas/complex_schema.graphql",
    error_type: MyError,
    with_idents: [WrongIdent]
);