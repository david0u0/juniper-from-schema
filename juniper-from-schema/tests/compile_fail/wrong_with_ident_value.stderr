error: proc macro panicked
  --> $DIR/wrong_with_ident_value.rs:6:1
   |
6  | / graphql_schema_from_file!(
7  | |     "../../../juniper-from-schema/tests/schemas/complex_schema.graphql",
8  | |     error_type: MyError,
9  | |     with_idents: [WrongIdent]
10 | | );
   | |__^
   |
   = help: message: 
           
           [91merror[0m: Identity `WrongIdent` is not found in schema
           
           Each value in `with_ident` must be in the schema.
           
           
           aborting due to previous error
